staging/operator-lifecycle-manager/pkg/lib/ownerutil/util.go

```diff
@@ -272,6 +272,11 @@ func CSVOwnerSelector(owner *operatorsv1alpha1.ClusterServiceVersion) labels.Sel
 	return labels.SelectorFromSet(OwnerLabel(owner, operatorsv1alpha1.ClusterServiceVersionKind))
 }
 
+// OperatorGroupOwnerSelector returns a label selector to find generated objects owned by owner
+func OperatorGroupOwnerSelector(owner *operatorsv1.OperatorGroup) labels.Selector {
+	return labels.SelectorFromSet(OwnerLabel(owner, operatorsv1.OperatorGroupKind))
+}
+
```

Comment:
I checked this PR and the master branch upstream and this function doesn't seem to be used anywhere.

Reply:
https://github.com/operator-framework/operator-lifecycle-manager/issues/3047

Reply:
I can remove this, too